src/main/java/org/jqassistant/contrib/plugin/php/scanner/parser/Helper.java

```diff
@@ -245,19 +254,30 @@ private String function_getFullQualifiedName(String Name, PHPNamespaceDescriptor
         return "FUNCTION|" + fullname;
     }
     
+    /**
+     * create new property
+     * @return PHPPropertyDescriptor
+     */
     public PHPPropertyDescriptor getProperty(){
         PHPPropertyDescriptor phpProperty = store.create(PHPPropertyDescriptor.class);
         phpProperty.setVisibility(VisibilityModifier.DEFAULT);
         phpProperty.setStatic(Boolean.FALSE);
+        phpProperty.setConstant(Boolean.FALSE);
         
         return phpProperty;
     }
     
+    /**
+     * create new function parameter
+     * @param idx
+     * @param name
+     * @return PHPFunctionParameterDescriptor
+     */
      public PHPFunctionParameterDescriptor getFunctionParameter(Integer idx, String name){
         PHPFunctionParameterDescriptor phpFunctionParameter = store.create(PHPFunctionParameterDescriptor.class);
         phpFunctionParameter.setIndex(idx);
         phpFunctionParameter.setName(name);
-        System.err.println("ADD FunctionParameter: " + idx + " " + name);
+        logInfo("ADD FunctionParameter: " + idx + " " + name);
```

Comment:
And thank you for fixing this logging level stuff :)

It is strange that the author used `System.err`.